mobile-widgets/qml/DiveDetails.qml

```diff
@@ -272,7 +267,10 @@ Kirigami.Page {
 		cylinderIndex4 = currentItem.modelData.dive.cylinderList.indexOf(usedCyl[4])
 		rating = currentItem.modelData.dive.rating
 		visibility = currentItem.modelData.dive.visibility
-
+		if (!currentItem.modelData) {
+			console.log("DiveDetails trying to access undefined currentItem.modelData")
+			return
+		}
```

Comment:
Why move this check to the end, after already using _currentItem.modelData_? So, this check will have no real effect if _currentItem.modelData_ is undefined.